### PR TITLE
Feature/card queries

### LINKED ISSUE #1: As a user, when I visit the application, I'm greeted by a landing page consisting of 1 album elements.
- [x] 1 albums
- [x] Header
- [x] Sidebar/Navigation
- [x] Test Passing

### PR DESCRIPTION
**Pull request by Max9545**
# Selector FE PR

## What does this merge do? Check all that apply.
- [x] Adds a new feature
- [ ] Improves an existing feature
- [x] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## Summary of changes

- Make tests and fixtures for album cards and pass all tests

## Notes for reviewer (any known bugs, areas of interest, etc.)

- Can change info in fixture to make sure stub is working

## Next Steps (Future Iterations)

- Style more to match comp

### CODE REVIEW

File: cypress/integration/examples/albumCards.spec.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+context('Album Cards', () => {
+  beforeEach(() => {
+    cy
+    .fixture('freakOut.js').as('freakOut')
```
Comment (monshan) [inline cypress/integration/examples/albumCards.spec.js]: Think we can remove this line as the fixture is being defined on line 7

File: cypress/integration/examples/albumCards.spec.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    cy
+    .get('[data-cy=genre]').should('exist')
+    .should('contain', 'Electronic')
+    .should('contain', 'Rock')
+  })
+
+  it('Should have a link to purchase the album on discogs', () => {
+    cy
+    .get('[data-cy=discogs-link]').should('exist')
+    .should('contain', 'Purchase on Discogs')
```
Comment (monshan) [inline cypress/integration/examples/albumCards.spec.js:46]: Should we also check the attr a href value to ensure they are being directed to the expected address

File: cypress/integration/examples/albumCards.spec.js (hunk 1 above)
Comment (Max9545) [reply cypress/integration/examples/albumCards.spec.js]: done

File: src/Components/AlbumCard/AlbumCard.js [hunk 3]
```diff
@@ -7,15 +7,15 @@ function AlbumCard({ coverImage, title, name, year, genres }) {
   }
 
   return (
-    <div className='card'>
+    <div className='card' data-cy='card'>
       <img className='cover' data-cy='cover'src={coverImage}/>
       <h2 className='title' data-cy='title'>{title}</h2>
-      <h3 className='artist' dat-cy='artist'>{name}</h3>
+      <h3 className='artist' data-cy='artist'>{name}</h3>
```
Comment (monshan) [inline src/Components/AlbumCard/AlbumCard.js:13]: Nice catch

**Review by monshan — CHANGES_REQUESTED** [review]
Happy we were able to get testing done with a functioning integration, just make one addition and it can be merged

1. Test for the href address on discogs button

File: cypress/integration/examples/albumCards.spec.js (hunk 2 above)
Comment (Max9545) [reply cypress/integration/examples/albumCards.spec.js:46]: we will need this a link to populate this button with from BE, message sent

**Pull request merged**